storage/data-plane/src/test/java/com/microsoft/azure/storage/AppendBlobAPITest.groovy

```diff
@@ -306,4 +296,176 @@ class AppendBlobAPITest extends APISpec {
         then:
         notThrown(RuntimeException)
     }
+
+
+    // Grouping the below tests from PageBlobAPITest and BlockBlobAPITest together to prevent Concurrent modification exception.
+    @Unroll
+    def "Stage block illegal arguments"() {
```

Comment:
Following up on our discussion that we should fix the root cause of why these tests are failing when in separate test classes rather than move them. I'm fine with leaving this for now and hoping that merging in v11 with overwrite this change + fix the issue because of the fixes in some dependencies, but I don't want to keep these tests here beyond the merge of v11

Reply:
Yeah, 
with v11, I will bring the tests as they are and test them again in our dev ops and see if any issues come up.